acestep/ui/gradio/events/library_handlers.py

```
@@ -0,0 +1,201 @@
+
+def rating_stars(rating: int) -> str:
+    """Return a star-string representation for a 0-5 rating."""
+    if not rating:
+        return "—"
+    return "★" * int(rating) + "☆" * (5 - int(rating))
+
+
+def _format_date(ts: int) -> str:
+    try:
+        return datetime.fromtimestamp(ts).strftime("%Y-%m-%d %H:%M")
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)